examples/rvc-app/README.md

```diff
@@ -0,0 +1,132 @@
+
+**Note:** If the testing environment has not been commissioned with the RVC app,
+use chip-tool to switch on the commissioning window
+`chip-tool pairing open-commissioning-window`, and add the following flags ta
+the `--script-args` above.
+`--commissioning-method on-network --discriminator XXXX --passcode XXXX`.
+
+Below are the PICS files and PIXIT definitions required for the different python
+tests.
+
+### RvcCleanMode cluster
+
+PICS: `examples/rvc-app/rvc-common/pics/RVC_Clean_Mode_Cluster_Test_Plan.txt`
+
+#### TC_RVCCLEANM_1_2.py
+
+PIXIT: `PIXIT_ENDPOINT:1`  
+Example command:
+`./scripts/tests/run_python_test.py --script src/python_testing/TC_RVCCLEANM_1_2.py --script-args "--storage-path admin_storage.json --PICS examples/rvc-app/rvc-common/pics/RVC_Clean_Mode_Cluster_Test_Plan.txt --int-arg PIXIT_ENDPOINT:1"`
+
+#### TC_RVCCLEANM_2_1.py
+
+PIXIT:
+`PIXIT_ENDPOINT:1 PIXIT.RVCCLEANM.MODE_CHANGE_FAIL:1 PIXIT.RVCCLEANM.MODE_CHANGE_OK:2`  
+Example command:
+`/scripts/tests/run_python_test.py --script src/python_testing/TC_RVCCLEANM_2_1.py --script-args "--storage-path admin_storage.json --PICS examples/rvc-app/rvc-common/pics/RVC_Clean_Mode_Cluster_Test_Plan.txt --int-arg PIXIT_ENDPOINT:1 PIXIT.RVCCLEANM.MODE_CHANGE_FAIL:1 PIXIT.RVCCLEANM.MODE_CHANGE_OK:2"`
+
+When asked "Manually put the device in a state from which it will FAIL to
+transition to mode 1", set the RvcRunMode to 1.
+`chip-tool rvcrunmode change-to-mode 1`
+
+When asked "Manually put the device in a state from which it will SUCCESSFULLY
+transition to mode 2", set the RvcRunMode to 0.
+`chip-tool rvcrunmode change-to-mode 0`
+
+#### TC_RVCCLEANM_3_2.py
+
+This is not applicable because the RVC device does not support the StartUpMode
+attribute.
+
+### RvcRunMode cluster
+
+PICS: `examples/rvc-app/rvc-common/pics/RVC_Run_Mode_Cluster_Test_Plan.txt`
+
+#### TC_RVCRUNM_1_2.py
+
+PIXIT: `PIXIT_ENDPOINT:1` Example command:
+`./scripts/tests/run_python_test.py --script src/python_testing/TC_RVCRUNM_1_2.py --script-args "--storage-path admin_storage.json --PICS examples/rvc-app/rvc-common/pics/RVC_Run_Mode_Cluster_Test_Plan.txt --int-arg PIXIT_ENDPOINT:1"`
+
+#### TC_RVCRUNM_2_1.py
+
+PIXIT:
+`PIXIT_ENDPOINT:1 PIXIT.RVCRUNM.MODE_CHANGE_FAIL:2 PIXIT.RVCRUNM.MODE_CHANGE_OK:0`  
+Example command:
+`./scripts/tests/run_python_test.py --script src/python_testing/TC_RVCRUNM_2_1.py --script-args "--storage-path admin_storage.json --PICS examples/rvc-app/rvc-common/pics/RVC_Run_Mode_Cluster_Test_Plan.txt --int-arg PIXIT_ENDPOINT:1 PIXIT.RVCRUNM.MODE_CHANGE_FAIL:2 PIXIT.RVCRUNM.MODE_CHANGE_OK:0"`
+
+When asked "Manually put the device in a state from which it will FAIL to
+transition to mode 2", set the RvcRunMode to 1.
+`chip-tool rvcrunmode change-to-mode 1`
+
+When asked "Manually put the device in a state from which it will SUCCESSFULLY
+transition to mode 0", do nothing.
+
+#### TC_RVCRUNM_3_2.py
+
+This is not applicable because the RVC device does not support the StartUpMode
```

Comment:
The RVC device spec doesn't mention that. However, it does say that the OnOff dependency (see the feature map) is not supported for the RVC Run and Clean clusters. So TC 3.2 applies, but 3.1 and 3.3 don't.

Reply:
This particular RVC device doesn't support the StartUpMode though. So for this app TC 3.2 is still not applicable. We can change this in a future PR.